convergence_acoustic_elastic/convergence_ocean/rigid.yaml

```diff
@@ -0,0 +1,3 @@
+!ConstantMap
+    map:
+      map_u_u: -1.0
```

Comment:
add a comment about this parameter.
Why is the file named rigid?

Reply:
It's an undocumented feature - it creates a rigid boundary condition (=wall).
This feature will be removed at some point and thus I do not want people to rely on it. 
I'll add a comment in the parameters.par though.